log4j-api/src/main/java/org/apache/logging/log4j/Level.java

```diff
@@ -259,15 +258,23 @@ public String toString() {
      * @throws java.lang.IllegalArgumentException if the name is null or intValue is less than zero.
      */
     public static Level forName(final String name, final int intValue) {
-        final Level level = LEVELS.get(name);
```

Comment:
Above we use Strings.isEmpty(name), shouldn't we use the same here?

Reply:
Yes, that make sense, although the main reason for this check is to prevent an NPE in the `ConcurrentHashMap`.